std/variant.d

```diff
@@ -2239,19 +2289,18 @@ private auto visitImpl(bool Strict, VariantType, Handler...)(VariantType variant
 
             static if (dgIdx == -1)
             {
-                static if (Strict)
+                static if (HandlerOverloadMap.generalFuncIdx >= 0)
+                    return Handler[ HandlerOverloadMap.generalFuncIdx ](*ptr);
```

Comment:
Not sure whether we can automatically enforce this. There seem to be many examples like this one:

```
std/algorithm/iteration.d
4779:    return (((cast(F) r[ 0] + r[ 1]) + (cast(F) r[ 2] + r[ 3]))
4780:          + ((cast(F) r[ 4] + r[ 5]) + (cast(F) r[ 6] + r[ 7])))
4781:         + (((cast(F) r[ 8] + r[ 9]) + (cast(F) r[10] + r[11]))
```

(use e.g. `grep -ir "\[ [[:alnum:]]\]" *` for the full list)